### PR TITLE
Docs hex colors

### PR DESCRIPTION
**Pull request by anitavincent**
#### What is the purpose of this pull request?
Add hex values to colors on docs.

#### What problem is this solving?
Docs improvement

#### How should this be manually tested?
Check built docs + yarn test

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

**Review by Lenderson — APPROVED** [review]

File: packages/admin-ui-docs/src/components/utils.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { hsl as getHexFromHsl, parseToHsl, parseToRgb } from 'polished'
+import type { RgbaColor } from 'polished/lib/types/color'
```
Comment (marcelovicentegc) [inline packages/admin-ui-docs/src/components/utils.ts:2]: Could this be imported directly from `polished` instead?

```suggestion
import type { RgbaColor } from 'polished'
```

File: packages/admin-ui-docs/src/components/utils.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export function replaceHslForHex(
+  original: string,
+  props?: { keepBothValues: boolean }
+) {
+  const { keepBothValues } = props || { keepBothValues: false }
+  const hslaRegex = /(hsl)a?\([^)]*\)/gi
+
+  const newString = original.replace(hslaRegex, (match) => {
+    return !keepBothValues ? hslaToHex(match) : `${match} ${hslaToHex(match)}`
```
Comment (marcelovicentegc) [inline packages/admin-ui-docs/src/components/utils.ts:34]:
```suggestion
    return keepBothValues ? `${match} ${hslaToHex(match)}` : hslaToHex(match)
```

**Review by marcelovicentegc — APPROVED** [review]
LGTM! Just left a question and one suggestion 😉

File: packages/admin-ui-docs/src/components/utils.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  })
```
Comment (Lenderson) [inline packages/admin-ui-docs/src/components/utils.ts:35]:
```suggestion
  const newString = original.replace(hslaRegex, (match) =>
      keepBothValues ? `${match} ${hslaToHex(match)}` : hslaToHex(match)
  )
```
```

**Review by matheusps — COMMENTED** [review]
I think this should go to the next branch

**Pull request merged**